docker/build_jar.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I am confused on the coupling between the `terra-github-workflows` repo and Rawls. It appears there is a Rawls specific workflow living in that repo that depends on this script being present. Why not have that workflow live in the Rawls repo?

Reply:
The reasons are:

- We want to separate pipeline-related stuff from the Rawls repo (same as for Sam, Leo, and Orch). 
- The workflows in `terra-github-workflows` repo are parameterized to make them reusable across different use cases. So for example, in this PR, we build the `rawls-build-tag-publish-and-run-tests.yaml` workflow to show how to dispatch Rawls swat tests against BEE environment. The swat tests are simply plugged into the workflow by passing in the appropriate parameterss.
- The pipeline has separate security requirements from Rawls. So like `terra-github-workflows` have access to certain repositories for publishing artifacts and this can be managed from the `terra-github-workflows` repo.
- We have the separate `build_jar.sh` and `clean_install.sh` just for doing a clean build and no tests at all. Because now with the new IRWG pipeline, dev can simply plugged in the kind of tests (e.g. smoke tests) in the order they want the workflow to perform. These shell scripts are specific to the Rawls stack so they live in the Rawls repo.